.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: cpp
+compiler: gcc
+dist: trusty
+sudo: true
+env:
+  global:
+    # The next declaration is the encrypted COVERITY_SCAN_TOKEN, created
+    #   via the "travis encrypt" command using the project repo's public key
+    - secure: "nCQtEBd5gQ9zirHNQVpzB0Q8aVKmMuTrlHfZCK5ZOpFItD9zP0YwqUTcw6y8T+14CceY6Go/D+fgeMYR3QmYc2CW0vXMwgBYOFQuPYEef7455ZPdt6wdr4lnyP/B+fj4cWZ4WhV+hMigl2Ly4xaFH6msm+PlIHOdjdaXo3ko0LI="
```

Comment:
This is the same token that is used in libuavcan, it won't work here because it is specific to the public key of that repo. I think we can safely get by without automated Coverity scans here for now.

Reply:
Erp. I thought this was for Travis. I'll remove. 

How do we set this up for Travis CI builds?

Reply:
Just remove this section entirely, it's not needed at all.

Reply:
Okay. Removed.